host-spec/c06-consensus.tm

```diff
@@ -302,28 +369,47 @@
 
   For any slot <math|i> in epoch <math|n> where <math|d\<less\>\<tau\>>, the
   block producer is required to produce a block. For the definitions of
-  <name|Epoch-Randomness<math|>> and <em|<name|VRF>> functions, see Algorithm
-  <reference|algo-epoch-randomness> and Section <reference|sect-vrf>
+  <name|Epoch-Randomness<math|>> and <em|<name|VRF>> functions, see Section
+  <reference|sect-epoch-randomness> and Section <reference|sect-vrf>
   respectively.
 
   <subsection|Slot Number Calculation>
 
-  It is essential for a block producer to calculate and validate the slot
-  number at a certain point in time. Slots are dividing the time continuum in
-  an overlapping interval. At a given time, the block producer should be able
-  to determine the set of slots which can be associated to a valid block
-  generated at that time. We formalize the notion of validity in the
-  following definitions:
+  It is imperative for the security of the network that each block producer
+  is able to correctly determine the set of slots which can be associated to
+  a valid block generated at a given time without relying on any external
+  clock source (e.g. through the <with|font-shape|italic|Network Time
+  Protocol> or the <with|font-shape|italic|Global Positioning System>). To
+  stay in synchronization each producer is therefore required to periodically
+  estimate its local clock offset in relation to the rest of the network.\ 
+
+  This estimation depends on the two fixed parameters
+  <with|font-series|bold|<math|k>> (def. <reference|defn-prunned-best>) and
+  <math|<with|font-series|bold|s<rsub|cq>>> (def.
+  <reference|defn-chain-quality>). These are choosen based on the results of
+  simulation parameterized to meet certain security bounds, currently
```

Comment:
These are choosen based on the results of simulation parameterized-> These are chosen based on the results of formal security analysis